enumerable_methods.rb.rb

```diff
@@ -0,0 +1,100 @@
+module Enumerable
+  def my_each
+    array = self
+    array.size.times { |i| yield array[i] }
+    array
+  end
+
+  def my_each_with_index
+    array = self
+    array.length.times { |i| yield array[i], i }
+    array
+  end
+
+  def my_select
+    array = self
+    my_select_array = []
+    array.my_each { |item| my_select_array << item if yield(item) }
+    my_select_array
+  end
```

Comment:
- [ ] should have working `my_select` identical and returns the same thing as ruby's [`select`](https://ruby-doc.org/core-2.6.4/Enumerable.html#method-i-select). ([Screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/advanced_building_blocks_enumerable/my_select.png) from [Odin](https://www.theodinproject.com/courses/ruby-programming/lessons/advanced-building-blocks#assignment-2))

Yours is not Returning an enumerator when no block is given.